packages/types/src/modules.ts

```diff
@@ -5,6 +5,7 @@ export enum ModuleKind {
   Mint = 'mint',
   Wallet = 'wallet',
   Meta = 'meta',
+  Unknown = 'unknown',
```

Comment:
I worry a bit about this enum, does this mean we will encounter errors when encountering other modules than the ones listed here?

Reply:
Going to land this and open as a separate issue for unknown module handling